example_scenes/advanced_tex_fonts.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
I don't think example scenes should have the import. Please remove them from all example scenes.